cloud/endagaweb/templates/dashboard/index.html

```diff
@@ -23,6 +23,54 @@
     display: none;
   }
 </style>
+<html lang="en">
+<head>
+  <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+  <script>
+
+$(document).ready(function(){
+$("#dialog").modal('show');
+});
+
+ </script>
+</head>
+<body>
+{% if messages %}
+<div class='modal fade' id='dialog'>
+  <div class='modal-dialog'>
+    <div class='modal-content'>
+      <div class='modal-header'>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <h4 class='modal-title'>
+          Password Expiry Status
+        </h4>
+      </div>
+      <div class='modal-body'>
+        {% for message in messages %}
+        <p>
+          {% if message.text %} {% endif %} Hi {{ message }}
```

Comment:
I don't understand what this `{% if ... %} {% endif %}` block is doing, it looks like nothing?